CreeDictionary/CreeDictionary/paradigm/panes.py

```diff
@@ -16,23 +24,58 @@ class ParadigmTemplate:
     def __init__(self, panes: Iterable[Pane]):
         self._panes = tuple(panes)
 
+    @property
+    def max_num_columns(self):
+        """
+        What is the largest amount of columns necessary for this entire paradigm.
```

Comment:
You’re mostly calling this a number but In a couple places it’s called an amount; ‘number’ is more appropriate.
```suggestion
        How many columns are necessary for this entire paradigm?
```